src/v/config/configuration.cc

```diff
@@ -1545,7 +1545,15 @@ configuration::configuration()
       "Maximum capacity of rate limit accumulation"
       "in controller configuration operations limit",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
-      std::nullopt) {}
+      std::nullopt)
+  , crash_loop_max_limit(
+      *this,
+      "crash_loop_max_limit",
+      "Maximum consecutive crashes (unclean shutdowns) allowed after which "
+      "operator intervention is needed to startup the broker.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      std::numeric_limits<uint32_t>::max(),
+      {.min = 1, .max = std::numeric_limits<uint32_t>::max()}) {}
```

Comment:
Rather than making this an integer that takes a very high value to disable, let's use a std::optional, so that we can clearly know when the feature is disabled (nullopt)

Reply:
Done.